metricbeat/module/beat/beat_test.go

```diff
@@ -68,6 +67,6 @@ func TestFetchURI(t *testing.T) {
 		u, err := url.Parse(tc.orig)
 		require.NoError(t, err)
 		got := fetchURI(u, tc.path)
-		assert.Equal(t, tc.want, got)
+		require.Equal(t, tc.want, got)
```

Comment:
this was `assert` and it became `require` which results in not checking the remaining test cases. If you wanna go with require you need to add t.Run(...) for each test case?